exporter/collector/metrics_integration_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Google LLC
```

Comment:
Should this go in a subdirectory somewhere? Inside `internal/` maybe?

Reply:
I think it's fine here until you want to use it from tests in multiple packages.